css-transitions-2/Overview.bs

```diff
@@ -72,12 +72,29 @@ spec:web-animations-1; type:dfn;
 
 ## The 'transition-property' Property ## {#transition-name-property}
 
+ISSUE: Insert text from [[css-transitions-1#transition-property-property]]
+and backport the following paragraph.
+
 Although the 'transition-property' may specify shorthand properties and the
 ''all'' keyword, individual transitions are generated for each longhand
 sub-property that is animatable. The <dfn>expanded transition property
```

Comment:
I think this looks like a previous edit that may have been intended to partly fix this issue.  However, I think this paragraph needs modification to match the resolution -- shorthands should consider all "transitionable" or "animatable" properties (I think), whereas "all" should reject both properties that are "not animatable" or whose animation type is "discrete".

On the other hand, I think perhaps level 2 should really get rid of the "transitionable" definition since it just means not "not animatable", and just use "not animatable" as needed.

Reply:
I'm just adding an issue here to clarify that this section is incomplete; I'm not changing the existing paragraph that follows the issue, which is defining a term used elsewhere in the spec. I don't see any reason why that paragraph would need edits here.

Reply:
Yeah, I know you weren't editing it... but I was suggesting that you should.  That existing paragraph (and I didn't check why it was inserted... but I'm somewhat curious) states pretty clearly that `all` and shorthands work the same way.

Reply:
It says that each sub-property of a shorthand of the `all` keyword generates its own transition. But we also say that the `all` keyword doesn't map to discretely-animatable properties (unlike the `all` shorthand itself), so discretely-animatable longhands aren't part of `all` and therefore aren't addressed by this paragraph afaict.